index.js

```diff
@@ -2673,17 +2682,19 @@
 
 }));
 
-//. [Apply]:        https://github.com/fantasyland/fantasy-land#apply
-//. [Extend]:       https://github.com/fantasyland/fantasy-land#extend
-//. [Functor]:      https://github.com/fantasyland/fantasy-land#functor
-//. [Monad]:        https://github.com/fantasyland/fantasy-land#monad
-//. [Monoid]:       https://github.com/fantasyland/fantasy-land#monoid
-//. [Traversable]:  https://github.com/fantasyland/fantasy-land#traversable
-//. [R.equals]:     http://ramdajs.com/docs/#equals
-//. [R.map]:        http://ramdajs.com/docs/#map
-//. [R.type]:       http://ramdajs.com/docs/#type
-//. [Ramda]:        http://ramdajs.com/
-//. [RegExp]:       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
-//. [RegexFlags]:   https://github.com/plaid/sanctuary-def#regexflags
-//. [Semigroup]:    https://github.com/fantasyland/fantasy-land#semigroup
-//. [parseInt]:     https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
+//. [Apply]:          https://github.com/fantasyland/fantasy-land#apply
+//. [Extend]:         https://github.com/fantasyland/fantasy-land#extend
+//. [Functor]:        https://github.com/fantasyland/fantasy-land#functor
+//. [Monad]:          https://github.com/fantasyland/fantasy-land#monad
+//. [Monoid]:         https://github.com/fantasyland/fantasy-land#monoid
+//. [Traversable]:    https://github.com/fantasyland/fantasy-land#traversable
+//. [R.equals]:       http://ramdajs.com/docs/#equals
+//. [R.map]:          http://ramdajs.com/docs/#map
+//. [R.type]:         http://ramdajs.com/docs/#type
+//. [Ramda]:          http://ramdajs.com/
+//. [RegExp]:         https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
+//. [RegexFlags]:     https://github.com/plaid/sanctuary-def#regexflags
+//. [sanctuary-def]:  https://github.com/plaid/sanctuary-def
+//. [BinaryType]:     https://github.com/plaid/sanctuary-def#binarytype
+//. [Semigroup]:      https://github.com/fantasyland/fantasy-land#semigroup
+//. [parseInt]:       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
```

Comment:
Let's keep these links lexicographically ordered by identifier. In Vim, I type `vip:sort u` to achieve this. :)